TESTING_WITH_IAF-TEST.md

```diff
@@ -66,7 +67,7 @@ See the Eclipse instructions for an important note about the `log.dir` setting.
 ## 4. Select database
 
 The ibis-adapterframeworkt-test project supports multiple databases. By default, an H2 local database is used.
-Docker projects for a number of other DMBSes are provided in iaf/docker/dbms. To use one of the provided databases, run the `rebuild.bat` script in the corresponding directory. (requires Docker to be installed on your machine). To configure the ibis-adapterframeworkt-test application, set in the catalina.properties or the VM Options the property `jdbc.dbms.default` to `oracle`, `mssql`, `mysql`, `mariadb` or `postgres`.
+Docker projects for a number of other DMBSes are provided in GitHub project https://github.com/ibissource/iaf-ci-images. To use one of the provided databases, run the `rebuild.bat` script in the corresponding directory. (requires Docker to be installed on your machine). To configure the ibis-adapterframeworkt-test application, set in the catalina.properties or the VM Options the property `jdbc.dbms.default` to `oracle`, `mssql`, `mysql`, `mariadb` or `postgres`. If you have no preference about the database you are working with, then do not choose Oracle. It is complicated to build a docker image for an Oracle database that can be used to test against.
```

Comment:
Dit voegt helemaal niets toe. Als je perse Oracle wilt gebruikt ga ik er van uit dat je ook weet waarom en hoe Oracle werkt. Sterker nog, dan heb je waarschijnlijk al een database draaien.
Het framework start ook prima (standaard) met H2 op. Dit is allemaal optioneel, dan moet je niet zeggen 'if you have no preference ...'. 
```suggestion
Docker projects for a number of other DMBSes are provided in GitHub project https://github.com/ibissource/iaf-ci-images. To use one of the provided databases, run the `rebuild.bat` script in the corresponding directory. (requires Docker to be installed on your machine). To configure the ibis-adapterframeworkt-test application, set in the catalina.properties or the VM Options the property `jdbc.dbms.default` to `oracle`, `mssql`, `mysql`, `mariadb` or `postgres`.
```